ims/company/device_security.md

```diff
@@ -0,0 +1,57 @@
+# Device Security
+
+This is general advice on device security. For the purposes of this section, "device" not only applies to the laptop Made Tech will issue you with when you join the company, but also any personal devices you use to access Made Tech systems. This includes, but is not limited to, phones, tablets, etc. 
+
+Please consult our BYOD policy and our AUP policy for more information (linked below). 
+
+## Laptop Setup
+
+This section details policy around Made Tech issued devices. If you are using your own device at Made Tech please consult our BYOD policy (linked below)
+
+All Made Tech devices are set up with separate admin and user profiles, unless otherwise agreed with the relevant IMS governance team or leadership. For more information on this please see our Standard Builds policy (linked below).
```

Comment:
Let's just link here too.

Reply:
See "Reference-style links" in this markdown documentation for a way of making multiple links to the same content in a single document easier to manage: https://daringfireball.net/projects/markdown/syntax#link

Reply:
That's a bit of a scary read for a non-tech person. 😬 
I will try and puzzle it out and make it work.